src/utils/temp-cache.js

```diff
@@ -79,19 +70,15 @@ export class TempCache {
    * @return {T} Returns payload back
    */
   put(key, payload) {
-    return (this.maybeInitialize_()[key] = payload);
+    this.maybeSetFlushTimeout_();
+    return (this.cache_[key] = payload);
```

Comment:
Nit: break this into two lines. The minifier will do the magic.

Reply:
Nice.